src/components/productdisplayitem.main.jsx

```diff
@@ -460,42 +520,65 @@ class ProductDisplayItemMain extends React.Component {
     );
   }
 
+  renderPowerReviewsElements() {
```

Comment:
*Suggestion*
This looks like pieces can be brought out into it's own `powerreviews` component and referenced wherever needed. What do you think?

Reply:
Here's a sample of how it can be done (previously done for another component):
https://github.com/elasticpath/react-pwa-reference-storefront/blob/master/src/components/indirecommendations.main.jsx